.github/workflows/e2e-test.yml

```diff
@@ -1,19 +1,16 @@
-name: Integration Tests
```

Comment:
Since this flow configuration file contains GraphQL tests, is the name of the flow correct? Can we count GraphQL tests as E2E also?

Reply:
It was a mistake, thanks for noticing, fixed.